lib/prawn.js

```diff
@@ -32,4 +32,32 @@ Prawn.prototype.moveDown = function() {
   this.dir = 'down';
 };
 
+Prawn.prototype.changeDirectionRight = function() {
+  if (this.dir === 'up' || this.dir === 'down')
+  {this.moveRight()}
```

Comment:
I think we can refactor this into one function which checks all directions at the same time.  I am going to approve and merge this and we can discuss a refactor tomorrow.  I might be oversimplifying.  This is great though, good thinking.

+Prawn.prototype.changeDirection = function() {
 +  if (this.dir === 'up' || this.dir === 'down') {'right', 'left'}
 +  if (this.dir === 'left' || this.dir === 'right') {'up', 'down')